packages/next/src/generators/application/lib/add-linting.ts

```diff
@@ -36,6 +36,11 @@ export async function addLinting(
         // Turn off @next/next/no-html-link-for-pages since there is an issue with nextjs throwing linting errors
         // TODO(nicholas): remove after Vercel updates nextjs linter to only lint ["*.ts", "*.tsx", "*.js", "*.jsx"]
 
+        reactEslintJson.ignorePatterns = [
```

Comment:
Could this go into a `.eslintignore` file at the root?

Reply:
I don't think so since next uses the react `.eslintignore`  [here](https://github.com/nrwl/nx/blob/master/packages/next/src/generators/application/lib/add-linting.ts#L28).

Reply:
I think the pattern `.next/` is project specific, so probably not

Reply:
I think since `.eslintignore` isn't always present ,we can keep this just in the app's ignore folder.